examples/ci/Makefile

```diff
@@ -3,16 +3,13 @@ PROJECT = ci
 COMMANDS = cmd/test
 #CODECOV_TOKEN = your codecov token
 
-# Including ci Makefile
-MAKEFILE = Makefile.main
-CI_REPOSITORY = https://github.com/src-d/ci.git
-CI_FOLDER = .ci
-
 # If you need to build more than one dockerfile, you can do so like this:
 # DOCKERFILES = Dockerfile_filename1:repositoryname1 Dockerfile_filename2:repositoryname2 ...
 
+# Including ci Makefile
+CI_REPOSITORY = https://github.com/src-d/ci.git
```

Comment:
It may be useful to let the user override this as well, specially for debuggin purposes.